### PR TITLE
test(pods-communication) add tests for basic pod communication

### PR DESCRIPTION
**Pull request by juliangieseke**
Add test case which creates a pod with two containers and tests
if a http-client on one can access nginx on the other container

Closes DCOS_15051

### CODE REVIEW

**Comment by jfurrow** [comment]
This test (as well as one other) are failing for me:
```
✘ Services Pods Create a pod with environment variable
✘ Services Pods should be able to communicate with eachother
```

**Comment by juliangieseke** [comment]
if not already deleted, can you please send me your results folder? i'll have a look at it tomorrow.

**Comment by jfurrow** [comment]
Sure thing! [results.zip](https://github.com/dcos/dcos-ui/files/1029905/results.zip)

File: system-tests/services/test-pods-communication.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+require("../_support/utils/ServicesUtil");
```
Comment (weblancaster) [inline system-tests/services/test-pods-communication.js]: Are we starting to do one test per file? `test-pods-communication.js` this is way too specific I believe. In this file we are testing we can edit the pod using the json format.

File: system-tests/services/test-pods-communication.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    it("should be able to communicate with eachother", function() {
+      const serviceName = "pods-communication-simple";
+
+      cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}/create`);
+
+      cy.contains("JSON Configuration").click();
+
+      cy.get("#brace-editor").setJSON(
+        `{
```
Comment (weblancaster) [inline system-tests/services/test-pods-communication.js]: can we format the entire object accordingly?

File: system-tests/services/test-pods-communication.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    afterEach(() => {
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("should be able to communicate with eachother", function() {
+      const serviceName = "pods-communication-simple";
+
+      cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}/create`);
```
Comment (weblancaster) [inline system-tests/services/test-pods-communication.js]: We can define this as setup section in this case `beforeEach`

File: system-tests/services/test-pods-communication.js (hunk 1 above)
Comment (weblancaster) [inline system-tests/services/test-pods-communication.js]: I believe we should add this as part of the `test-pod.js` file.

Comment (juliangieseke) [inline system-tests/services/test-pods-communication.js]: we wanted to group tests by feature, test-pod.js will be create, so communication will no fit into that one. we can add more tests regarding pod communication to this one later.

File: system-tests/services/test-pods-communication.js (hunk 3 above)
Comment (juliangieseke) [inline system-tests/services/test-pods-communication.js]: not sure, but when more tests are added we should consider it, yes.

File: system-tests/services/test-pods-communication.js (hunk 2 above)
Comment (juliangieseke) [inline system-tests/services/test-pods-communication.js]: what do you mean with that?

**Comment by tnguyen0** [comment]
Please see PR #2240.

**Pull request closed without merging**